pkg/doc/verifiable/credential_ldp_test.go

```diff
@@ -445,68 +447,88 @@ func TestExtraContextWithLDP(t *testing.T) {
 	r.NotNil(vcWithLdp)
 }
 
-//nolint:lll
 func TestParseCredentialFromLinkedDataProof_BbsBlsSignature2020(t *testing.T) {
 	r := require.New(t)
 
-	vcJSON := `
```

Comment:
We should still _also_ have tests based on a static test vectors.
@kdimak

Reply:
I will return them in the next stage - Mattr interop task

Reply:
Ok, thanks.